templates/template-creation-extension/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "template-creation-extension",
+  "version": "1.0.0",
+  "description": "[Description of extension]",
+  "author": "[Author's name]",
```

Comment:
Would be nice if the plugin could support template substitutions for something like this being `process.env.USER`